python/insights_ansible_playbook_lib/__init__.py

```diff
@@ -172,7 +173,7 @@ def verify_play(play: dict, gpg_key: bytes) -> bytes:
                 f"Play content failed to match its digest's signature: {serialized_play!r}."
             )
             raise GPGValidationError(
-                "Play digest does not match its signature",
+                "Play digest does not match its signature.",
```

Comment:
good change, although it's unrelated to the signer (and "pollutes" the history of this commit); what about a separate PR/commit?